### PR TITLE
add more context to readme file

### PR DESCRIPTION
**Pull request by ncclementi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
-# wwc test
-This is a repo to test git and github workflow
+# Women Who Code - DC Tet repository
```
Comment (ncclementi) [inline README.md]:
```suggestion
# Women Who Code - DC Test repository
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
-# wwc test
-This is a repo to test git and github workflow
+# Women Who Code - DC Test repository
+
+This is a repo to test git and github workflow, that was created during the itnro to OSS session
```
Comment (gsezenoktem) [inline README.md]:
```suggestion
This is a repo to test git and github workflow, that was created during the intro to OSS session
```

Comment (andiedoescode) [inline README.md]:
```suggestion
This is a repo to test git and github workflow, that was created during the intro to OSS session
```

**Review by angietechcafe — CHANGES_REQUESTED** [review]
This is a repo to test git and github workflow, that was created during the intro to OSS session

File: README.md [hunk 3]
```diff
@@ -1,2 +1,3 @@
-# wwc test
-This is a repo to test git and github workflow
+# Women Who Code - DC Test repository
+
+This is a repo to test git and github workflow, that was created during the intro to OSS session
```
Comment (ncclementi) [inline README.md:3]: let's test comment

File: README.md [hunk 4]
```diff
@@ -1,2 +1,3 @@
-# wwc test
-This is a repo to test git and github workflow
```
Comment (jgifford82) [inline README.md:2]: Test comment.

**Pull request merged**